pkg/templates/charts/toggle/cluster-backup/templates/hub-backup-pod.yaml

```diff
@@ -41,6 +41,9 @@ spec:
                   phase: Running
           remediationAction: inform
           severity: high
+          customMessage:
+            noncompliant: |-
+              Template is not compliant, cluster-backup-chart-clusterbackup pod is either not running in the open-cluster-management-backup namespace or is in error. 
```

Comment:
Not running or is in error almost seems like the same thing, but I suspect by not running you really mean doesn't exist.

Reply:
@gparvin correct, not running means doesn't exist